test/fixtures/3.0/ping/tslint.json

```diff
@@ -0,0 +1,4 @@
+{
+  "$schema": "http://json.schemastore.org/tslint",
+  "extends": ["@loopback/tslint-config/tslint.common.json"]
```

Comment:
Do we want to introduce tslint based linting to loopback-connector-swagger? Personally, I won't do any typescript-specific linting at all. Later on, we can configure `eslint` to run typescript checks, similarly to what you are proposing in loopback-next.